### ISSUE TITLE
Could not extract video info. Instance is likely blocked. 

### ISSUE THREAD

**Issue by johnwmail** [omarroth/invidious#877]
![image](https://user-images.githubusercontent.com/1752235/68300517-d6acd700-0095-11ea-8cdf-1e13443b6d37.png)

Could not extract video info. Instance is likely blocked. 
Why will that?
My IP blocked by youtube/google?

**Comment by elypter** [comment]
invidio.us is blocked by google. at least temporarily. that happed a few times recently. you can try other instances with this addon https://addons.mozilla.org/de/firefox/addon/invidition/

**Comment by omarroth** [comment]
See #811. The reason your instance is blocked is due to rate-limiting by Google. If you have IPv6 available on your machine, you can try `force_resolve: ipv6` as a potential fix.

**Comment by johnwmail** [comment]
Ok, but why invidio.us is not blocked?
it should be more traffic than my home server.
thanks.

**Comment by omarroth** [comment]
It does get blocked occasionally. 

Several smaller instances have also been getting blocked, as well as mobile apps such as NewPipe (see TeamNewPipe/NewPipe#2788). There isn't a specific known reason for why an IP may be getting rate-limited.

Closing in favor of #811.

**Comment by Tall-Lobster** [comment]
This issue has appeared again or I think it has within the past several days.

**Comment by thenanobel** [comment]
I'm facing this problem as well for last couple of days. To solve this issue temporarily, I've installed Invidious on my computer and watching videos through http://localhost:3000.
![Screenshot_2020-05-27 Error - Invidious](https://user-images.githubusercontent.com/9353047/82980717-f264d400-a00b-11ea-81f8-9c34c9751d86.png)

**Comment by woutfeys** [comment]
Me too...
